source/supported-hardware/upxtremei11.rst

```diff
@@ -0,0 +1,143 @@
+.. _upx11-board:
+
+|UPX11| Board
+---------------------
+
+The |UP Xtreme i11 board| (|UPX11|) is an x86 maker board based on Intel platform Tiger Lake UP3. The UP boards are used in IoT, industrial automation, digital signage areas, etc.
+
+Prerequisites
+^^^^^^^^^^^^^^^^
+
+|SPN| supports |UPX11| maker board. To start developing |SPN|, the following equipment, software and environments are required:
+
+* |UP Xtreme i11 board|
+* Custom SPI flashing cable (|instructions|).
+* |USB 2.0 pin header cable| for debug uart output. OR `Make your own <upx11-debug-uart-pinout_>`_.
+* DediProg SF100 or SF600 programmer
+* Linux host (see :ref:`running-on-linux` for details)
+* Internet access
+
+.. |UP Xtreme i11 board| raw:: html
+
+   <a href="https://up-shop.org/up-xtreme-i11-boards-series.html" target="_blank">UP Xtreme i11 board</a>
+
+.. |instructions| raw:: html
+
+   <a href="https://wiki.up-community.org/BIOS_chip_flashing_on_UP_Squared" target="_blank">instructions</a>
+
+.. |USB 2.0 pin header cable| raw:: html
+
+   <a href="https://up-shop.org/up-peripherals/110-usb-20-pin-header-cable.html" target="_blank">USB 2.0 pin header cable</a>
+
+
+Board Setup
+^^^^^^^^^^^^^^^^^
+
+.. image:: /images/upx11_setup.jpg
+   :width: 600
+   :alt: |UPX11| Board Setup
+   :align: center
+
+
+Before You Start
+^^^^^^^^^^^^^^^^^
+
+.. warning:: As you plan to reprogram the SPI flash, it's a good idea to backup the pre-installed BIOS image first.
+
+
+Boot the board and enter BIOS setup menu to get familiar with the board features and settings.
+
+.. _upx11-debug-uart-pinout:
+
+Early boot serial debug console can be reached via UART2 located on CN11 header on the |UPX11| board. Make sure you can observe serial output message running the factory BIOS first.
+
+.. note:: To make your own UART debug adapter by direct wiring, refer to CN11 Header Pinout for UART2:
+
+  +--------+--------------+
+  |  Pin   |    Signal    |
+  +--------+--------------+
+  |   8    |     GND      |
+  +--------+--------------+
+  |   9    |   UART_RX    |
+  +--------+--------------+
+  |   10   |   UART_TX    |
+  +--------+--------------+
+
+
+Building
+^^^^^^^^^^
+
+|UPX11| board is based on Intel |TGL|. To build::
+
+    python BuildLoader.py build tgl
+
+The output images are generated under ``Outputs`` directory.
+
+
+Stitching
+^^^^^^^^^^
+
+Stitch |SPN| images with factory BIOS image using the stitch tool::
+
+    python Platform/TigerlakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE_NAME> -s Outputs/tgl/Stitch_Components.zip -o <SBL_IFWI_IMAGE_NAME> -p 0xAA000204
+
+    <BIOS_IMAGE>     : Input file. Factory BIOS extracted from |UPX11| board.
```

Comment:
|UPX11| is not expanded in HTML.

Reply:
Fixed, and fixed in UP2 page too, same issue was carried over from there.